dockercfg.encrypted

```diff
@@ -0,0 +1,2 @@
+codeship:v2
```

Comment:
It might be good to check your version of `jet`. I think the newest version (2.15.0) will stamp this with `cloudbees:v1`.

Reply:
I'm running version 2.14.0, installed on Linux as directed by https://docs.cloudbees.com/docs/cloudbees-codeship/latest/pro-jet-cli/installation.  Where did you find 2.15.0?

Reply:
Run `jet update`. Their documentation is horribly out of date.